sdks/java/io/elasticsearch-tests/elasticsearch-tests-common/src/test/java/org/apache/beam/sdk/io/elasticsearch/ElasticsearchIOTestCommon.java

```diff
@@ -133,8 +133,7 @@ void testReadWithQuery() throws Exception {
             + "  \"query\": {\n"
             + "  \"match\" : {\n"
             + "    \"scientist\" : {\n"
-            + "      \"query\" : \"Einstein\",\n"
-            + "      \"type\" : \"boolean\"\n"
+            + "      \"query\" : \"Einstein\"\n"
```

Comment:
Remove to support both ES 5&6

Reply:
Nice! I see through the gree UTests that the test still work in ES5 and works in ES6. Thanks